.github/workflows/conda/macos_env.yml

```diff
@@ -0,0 +1,14 @@
+name: riscv_perf_model
```

Comment:
I copied `conda/environment_from_history.yml` and then made a couple of adjustments to create this file.